src/styles/components/statusbar.less

```diff
@@ -0,0 +1,49 @@
+.status-bar {
+  display: inline-flex;
+  background-color: @gray;
+  margin-right: @base-spacing-unit * 2/3;
+  height: @base-spacing-unit * 1/3;
+  width: 114px;
+  max-width: 100%;
+
+  .bar {
+    background-color: currentColor;
+    border-left: 1px solid @black;
+    &:first-child {
+      border-left: none;
+    }
+  }
+}
+
+.status-bar-column {
+  @media (min-width: @screen-medium-max) {
+    width: 330px;
+  }
+  width: 138px;
+
+}
+.status-bar-wrapper{
+  display: inline-block;
+  .color-1 {
+    color: @white;
+  }
+  .color-2 {
+    color: @yellow;
+  }
+}
+
+.unknown {
```

Comment:
1. These are defined globally. Could we either move them to a more fitting location or wrap them?
It also feels spammy to define them multiple places: https://github.com/dcos/dcos-ui/pull/61/files#diff-297ecda992ddfcaf45f3d9d4a59aa535R37
I think it would make more sense to define them in one of these two files.

2. I'm actually curious to why we have both style files. Isn't this just different specific situations of the same general style?